avocado/core/sysinfo.py

```diff
@@ -357,18 +366,21 @@ def __init__(self, basedir=None, log_packages=None, profiler=None):
                                            'profilers',
                                            key_type='str',
                                            default='')
-        self.profilers = genio.read_all_lines(profiler_file)
-
-        log.info('Profilers configured by file: %s', profiler_file)
-        log.info('Profilers declared: %s', self.profilers)
-        if not self.profilers:
-            self.profiler = False
-
-        if self.profiler is False:
+        self.profilers = []
+        if os.path.isfile(profiler_file):
+            self.profilers = genio.read_all_lines(profiler_file)
+            log.info('Profilers configured by file: %s', profiler_file)
+            log.info('Profilers declared: %s', self.profilers)
```

Comment:
This is outside the scope of this work, but, for symmetry, it'd be nice to either remove the "Profilers declared: " line, or add equivalent ones to commands and files.  Also, IMHO, "declared" doesn't sound like the best word here (and it's my fault).  I'd go with "defined" or "configured".